raster/src/main/scala/geotrellis/raster/geotiff/GeoTiffResampleRasterSource.scala

```diff
@@ -65,7 +65,7 @@ class GeoTiffResampleRasterSource(
     tiff.getClosestOverview(gridExtent.cellSize, strategy)
 
   def reprojection(targetCRS: CRS, resampleTarget: ResampleTarget = DefaultTarget, method: ResampleMethod = ResampleMethod.DEFAULT, strategy: OverviewStrategy = OverviewStrategy.DEFAULT): GeoTiffReprojectRasterSource =
-    new GeoTiffReprojectRasterSource(dataPath, targetCRS, resampleTarget, method, strategy, targetCellType = targetCellType) {
+    new GeoTiffReprojectRasterSource(dataPath, targetCRS, resampleTarget, method, strategy, targetCellType = targetCellType, baseTiff=Some(tiff)) {
```

Comment:
Could you add indentation? `baseTiff = Some(tiff)`

Reply:
:heavy_check_mark: